assets/vue/convos-connection-editor.vue

```diff
@@ -68,6 +71,7 @@ module.exports = {
   data: function() {
     return {
       advanced:         false,
+      on_connect:       "",
```

Comment:
Not in this line. Here the attribute `vue.onConnectCommands` contains a copy of the attribute `connection.on_connect_commands`. The reason for the copy is that I don't want to modify connection.on_connect_commands before you actually hit "Save"